src/jaxsim/parsers/rod/utils.py

```diff
@@ -206,3 +218,54 @@ def fibonacci_sphere(samples: int) -> npt.NDArray:
     return descriptions.SphereCollision(
         collidable_points=collidable_points, center=center_wrt_link
     )
+
+
+def create_mesh_collision(
+    collision: rod.Collision,
+    link_description: descriptions.LinkDescription,
+    method: MeshMappingMethods = MeshMappingMethods.VertexExtraction,
+    nsamples: int = 1000,
+) -> descriptions.MeshCollision:
+    file = pathlib.Path(resolve_local_uri(uri=collision.geometry.mesh.uri))
+    _file_type = file.suffix.replace(".", "")
+    mesh = trimesh.load_mesh(file, file_type=_file_type)
+
+    if mesh.is_empty:
+        logging.warning(f"Mesh {collision.geometry.mesh.uri} is empty, ignoring it")
+        return
```

Comment:
When can this happpen? The `mesh` variable is populated by `trimesh`. If the parsing fails, I'd expect that trimesh raises. If the library does not raise, I'd change the code as follows:

```suggestion
    if mesh.is_empty:
        raise RuntimeError(f"Failed to process '{file}' with trimesh")
```

Reply:
This happens when the ASCII-STL file is empty, e.g. for the root link mesh of ErgoCub when downloaded using `robot_descriptions` (probably a bug upstream)

Reply:
Unresolving

Reply:
Changed warning to exception